data/human/human missions.txt

```diff
@@ -3601,3 +3601,107 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+				
+mission "Street Swindle"
+	minor
+	source
+		government "Free Worlds" "Republic" "Syndicate"
+		attributes "urban"
+	on accept
+		payment -6000
+		fail
+	to offer
+		random < 4
+		credits >= 10000
+	on offer
+		conversation
+			`	You see a small crowd begining to gather on a street corner. A man in a flamboyant striped suit and matching hat calls "Come here! Come here! Come see the marvelous magician Mistress Marionette corral the crowded crossroads with her extravagant schemes!"`
+			choice
+				`	(Check it out.)`
+					goto observe
+				`	(Ignore it.)`
+					decline
+			label observe
+			`	You join them. A woman in similarly fancy garb sits behind a table, swiftly shuffling three cups before a hulking man. The announcer continues, "For our new guests, the chosen challenger's chance approaches! The current buy-in bet was 7000 credits; yes, we play with the big money here! Double or nothing, bet your bet: are you a player or a puppet?"`
+			`	The cups stop. After a brief pause, the man points to the cup on the right.`
+			choice
+				`	(Watch what happens.)`
+						goto continue
+				`	"You know these things are scams, right?"`
+			`	You try to warn him, but someone in the crowd shushes you. He doesn't hear.`
+				goto continue
+			label continue
+			`	"Are you certain in your choice?" the magician asks. "Once you..."`
+			`	"Yes, I'm certain," the man repeats. He draws 7000 credits and tosses them on the table. The magician lifts the cup, and a small red ball rolls from beneath. "And the first challenger proves to be a player!" the announcer says, mustering enthusiasm as the magician counts 14,000 credits to pay the man. "Let us find a new challenger! The next buy in is a mere 400 credits; if you haven't played before, now's the time to test your skill!"`
```

Comment:
```suggestion
			`	"Are you certain in your choice?" the magician asks. "Once you-"`
			`	"Yes, I'm certain," the man interrupts. He draws 7000 credits and tosses them on the table. The magician lifts the cup, and a small red ball rolls from beneath. "And the first challenger proves to be a player!" the announcer says, mustering enthusiasm as the magician counts 14,000 credits to pay the man. "Let us find a new challenger! The next buy in is a mere 400 credits; if you haven't played before, now's the time to test your skill!"`
```
Thought the interruption worked better for the man's `hulking` description rather than than the magician just trailing off.